tests/e2e/test_spyre_prompt_logprobs.py

```diff
@@ -75,27 +75,29 @@ def test_prompt_logprobs_must_be_enabled(monkeypatch: pytest.MonkeyPatch):
 
 @pytest.mark.cpu
 @pytest.mark.decoder
+@pytest.mark.parametrize("model", get_spyre_model_list())
 def test_prompt_logprobs_not_supported_with_cb(
-        monkeypatch: pytest.MonkeyPatch):
+        model: str, monkeypatch: pytest.MonkeyPatch):
     # Server shouldn't boot with both prompt logprobs and continuous batching
     # enabled
     monkeypatch.setenv("VLLM_SPYRE_ENABLE_PROMPT_LOGPROBS", "1")
     monkeypatch.setenv("VLLM_SPYRE_USE_CB", "1")
 
     with pytest.raises(ValueError, match="continuous batching"):
-        VllmConfig(model_config=ModelConfig(task="generate"))
+        VllmConfig(model_config=ModelConfig(model=model, task="generate"))
```

Comment:
This seems like a bug? No point in downloading the default facebook model for this test right?